examples/svelte-kit/src/stores/controls/ui.svelte

```diff
@@ -14,22 +14,23 @@
           data-testid={key}
           id={label}
           type="checkbox"
-          on:change={(e) => ctx.update((s) => ({ ...s, [key]: e.currentTarget?.checked }))}
-          checked={$ctx[key]}
+          on:change={(e) => context.update((s) => ({ ...s, [key]: e.currentTarget?.checked }))}
+          checked={$context[key]}
         />
         <label for={label}>{label}</label>
       </div>
     {:else if type === "string"}
       <div class="text">
-        <label style="margin-right:10px">{label}</label>
+        <label for={label} style="margin-right:10px">{label}</label>
         <input
+          id={label}
           data-testid={key}
           type="text"
           {placeholder}
-          value={$ctx[key]}
+          value={$context[key]}
           on:keydown={(e) => {
             if (e.key === "Enter") {
-              ctx.update((s) => ({ ...s, [key]: e.currentTarget.value }))
+              context.update((s) => ({ ...s, [key]: e.currentTarget.value }))
```

Comment:
```suggestion
              $context = { ...$context, [key]: e.currentTarget.value }
```

doesn't this work? I believe it should since we're in a Svelte file